bindings/gumjs/gumscript-runtime-dalvik.js

```diff
@@ -1643,13 +1711,32 @@
             return _api;
         }
 
-        var temporaryApi = {};
+        var temporaryApi = {addLocalReferenceFunc: null};
```

Comment:
Would be great to split this across three lines:

```JavaScript
var temporaryApi = {
    addLocalReferenceFunc: null
};
```

(either that or add a space on each end of `{}` for consistency).